docs/operator/cn/01-10-installation.md

```diff
@@ -0,0 +1,15 @@
+# Installing {Component Name}
+
+## Prerequisites
+
+<!--Describe the prerequisites for installing your service (if there are some).-->
```

Comment:
The following details are expected to present heres:
 - [components] What other components does this one depend on
 - What BTP integrations does this service is using/building upon